HGNC_Genomic_Files_Migration/README.md

```diff
@@ -0,0 +1,31 @@
+## HGNC Genomic Data Migration Tool
+
+This tool is intended to migrate the changes in entrez ids and hugo symbols based on HGNC in data files. The tool,
```

Comment:
"intended to update data files in accordance to <link_to_seed_DB_update>"

Reply:
Updated. Pointed the news release link to Gene Table Updates Readme for now. Will have to change the links once the location of the finalized/PR is merged.